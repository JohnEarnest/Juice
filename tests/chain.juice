// test for multiple chained if statements

int x = 0;
while(x < 5) {
	if      (x == 0) { print("first\n");   }
	else if (x == 1) { print("second\n");  }
	else if (x == 2) { print("third\n");   }
	else             { print("another\n"); }
	x = x + 1;
}