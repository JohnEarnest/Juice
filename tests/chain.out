first
second
third
another
another
